composer.json

```diff
@@ -30,8 +30,7 @@
         "ua-parser/uap-php": "^3.9.14"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.6",
-        "squizlabs/php_codesniffer": "^3.7",
+        "silverstripe/recipe-testing": "^3",
```

Comment:
Removing codesniffer because it's part of recipe-testing

Adding recipe-testing to get behat-extension which is required for the best test fix